src/mumble/GlobalShortcut_xdp.cpp

```diff
@@ -0,0 +1,189 @@
+// Copyright 2007-2022 Aleix Pol Gonzalez <aleixpol@kde.org>.
+// All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "GlobalShortcut_xdp.h"
+
+#include "Settings.h"
+#include "Global.h"
+#include "globalshortcuts_portal_interface.h"
+#include "portalsrequest_interface.h"
```

Comment:
Will these be autogenerated?

If so, they should probably become their own group of includes with a comment stating that these files are auto-generated.

Reply:
Yes, they're autogenerated. I've moved them to include separately.